bot/cogs/sticky.py

```diff
@@ -244,33 +275,45 @@ async def purge_sticky_message(self, interaction: Interaction) -> None:
                 res = await conn.fetch("SELECT * FROM sticky;")
                 data = [dict(row) for row in res]
                 for sticky in data:
-                    channel = await self.bot.fetch_channel(sticky["channel_id"])
-                    message = await channel.fetch_message(sticky["message_id"])
-                    await message.delete()
+                    try:
+                        channel = await self.bot.fetch_channel(sticky["channel_id"])
+                        message = await channel.fetch_message(sticky["message_id"])
+                        await message.delete()
+                    except discord.errors.NotFound:
+                        pass
```

Comment:
```suggestion
                    try:
                        channel = await self.bot.fetch_channel(sticky["channel_id"])
                        message = await channel.fetch_message(sticky["message_id"])
                        await message.delete()
                    except discord.errors.NotFound:
                        continue
```

Use `continue` statement in looping process to skip the iteration instead of `pass`. This is the best practice.